pkg/units/ffi.go

```diff
@@ -129,34 +155,43 @@ func (e *FFI) ValidateProgram(prog []uint64) (*fpb.ValidationResult, error) {
 }
 
 // RunProgram Runs the ebpf program and returns the execution results.
-func (e *FFI) RunProgram(executionRequest *fpb.ExecutionRequest) (*fpb.ExecutionResult, error) {
+func (e *FFI) RunEbpfProgram(executionRequest *fpb.ExecutionRequest) (*fpb.ExecutionResult, error) {
 	serializedProto, err := proto.Marshal(executionRequest)
 	if err != nil {
 		return nil, err
 	}
-	res := C.ffi_execute_bpf_program(unsafe.Pointer(&serializedProto[0]), C.ulong(len(serializedProto)))
+	res := C.ffi_execute_ebpf_program(unsafe.Pointer(&serializedProto[0]), C.ulong(len(serializedProto)))
 	return executionProtoFromStruct(&res)
 }
 
-// CreateMapArray creates an ebpf map of type array and returns its fd.
-// -1 means error.
-func (e *FFI) CreateMapArray(size uint64) int {
-	return int(C.ffi_create_bpf_map(C.ulong(size)))
-}
-
-// CloseFD closes the provided file descriptor.
-func (e *FFI) CloseFD(fd int) {
-	C.ffi_close_fd(C.int(fd))
-}
-
-// GetMapElements fetches the map elements of the given fd.
-func (e *FFI) GetMapElements(fd int, mapSize uint64) (*fpb.MapElements, error) {
-	res := C.ffi_get_map_elements(C.int(fd), C.ulong(mapSize))
-	return mapElementsProtoFromStruct(&res)
+// ---------- cBPF --------------
```

Comment:
[optional] would it be worth separating ebpf and cbpf in different files?

Reply:
Sounds good!

Reply:
The only reason why I think we should not do it is because both validate and run functions for cbpf and ebpf have helper functions in common, e.g. , `validationProtoFromStruct(...)` for validate and `executionProtoFromStruct(...)` for run